src/System.Text.Primitives/System/Text/Encoding/Utf16/Utf16TextEncoderLE.cs

```diff
@@ -152,6 +152,15 @@ public override bool TryEncode(string text, Span<byte> encodedBytes, out int byt
             return true;
         }
 
+        public override bool TryComputeEncodedBytes(ReadOnlySpan<char> utf16, out int bytesNeeded)
```

Comment:
There is a missing scenario for calculating the encoded bytes from a UTF-8 buffer to whatever encoding the encoder is targeting. The pattern for TryEncode/TryDecode was overloads for UTF-8, UTF-16, and UTF-32.